src/database/migrations/20190820173619-create-request.js

```diff
@@ -45,6 +45,10 @@ module.exports = {
         type: Sequelize.BOOLEAN,
         allowNull: false,
       },
+      multiCity: {
+        type: Sequelize.BOOLEAN,
+        allowNull: false,
```

Comment:
Since allow null is false, I think it would be nice to set the default value to false

Reply:
Thanks, will include it as stated, though it can be work fine without it as its already stated in the model.

Reply:
Nice one chief